ui/lib/kubernetes/addon/components/page/role/create-and-edit.js

```diff
@@ -135,20 +136,21 @@ export default class CreateAndEditRolePageComponent extends Component {
         this.args.model.name
       );
     } catch (error) {
-      const message = errorMessage(error, 'Error saving role. Please try again or contact support');
-      this.flashMessages.danger(message);
+      const message = error.errors ? error.errors.join('. ') : error.message;
```

Comment:
We should still use the `errorMessage` util which handles the formatting to keep things consistent.

Reply:
Sounds good! I will put that back in! :D